### PR TITLE
Add note on OOM risk for timevectors

### PR DESCRIPTION
**Pull request by JLockerman**
# Description

add note on oom risk in timevectors

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

File: timescaledb/how-to-guides/hyperfunctions/function-pipelines.md [hunk 1]
```diff
@@ -5,9 +5,13 @@ applying principles from functional programming and popular tools like Python
 Pandas, and PromQL.
 
 <highlight type="warning">
+
```
Comment (Loquacity) [inline timescaledb/how-to-guides/hyperfunctions/function-pipelines.md:8]:
```suggestion
```

File: timescaledb/how-to-guides/hyperfunctions/function-pipelines.md [hunk 2]
```diff
@@ -5,9 +5,13 @@ applying principles from functional programming and popular tools like Python
...
 
 <highlight type="warning">
+	
 Experimental features could have bugs! They might not be backwards compatible,
 and could be removed in future releases. Use these features at your own risk,
-and do not use any experimental features in production.
+and do not use any experimental features in production. Notably, `timevector()`
+_will_ materialize all its datapoints in memory, so using it on an excesively
+large dataset will run out of memory.
+
```
Comment (Loquacity) [inline timescaledb/how-to-guides/hyperfunctions/function-pipelines.md:14]:
```suggestion
and do not use any experimental features in production. 
</highlight>

<highlight type="important">
The `timevector()` function materializes all its data points in 
memory. This means that if you use it on a very large dataset, 
it runs out of memory. Do not use the `timevector` function 
on a large dataset, or in production.
```

Comment (Loquacity) [reply timescaledb/how-to-guides/hyperfunctions/function-pipelines.md:14]: The trick with admonitions is to keep them to a single topic, and to restate the problem in more than one way, especially if you're at important or warning level.

**Comment by davidkohn88** [comment]
These all look fine to me btw. I was sort of waiting to see @JLockerman if you wanted to work with @Loquacity's comments?

**Review by davidkohn88 — COMMENTED** [review]
LGTM, just address Lana's stuff and we're good...

**Review by Loquacity — APPROVED** [review]

File: timescaledb/how-to-guides/hyperfunctions/function-pipelines.md [hunk 3]
```diff
@@ -7,7 +7,14 @@ Pandas, and PromQL.
 <highlight type="warning">
 Experimental features could have bugs! They might not be backwards compatible,
 and could be removed in future releases. Use these features at your own risk,
-and do not use any experimental features in production.
+and do not use any experimental features in production. 
+</highlight>
+
```
Comment (davidkohn88) [inline timescaledb/how-to-guides/hyperfunctions/function-pipelines.md:12]: This, btw, is true of any of our functions requiring sorting as well...including at least `time_weight` and `counter_agg`...it seems a bit stark given that...

**Review by davidkohn88 — COMMENTED** [review]
Looks fine, though I think if this is that big a concern then it's a problem otherwise as well.

File: timescaledb/how-to-guides/hyperfunctions/function-pipelines.md (hunk 3 above)
Comment (Loquacity) [reply timescaledb/how-to-guides/hyperfunctions/function-pipelines.md:12]: We use this same comment in a lot of places. If you think we're missing it somewhere, we can add it in.

**Pull request merged**